### ISSUE TITLE
Review normaliziation layers

### ISSUE THREAD

**Issue by gpi-eaglys** [gpi-eaglys/homomorphic-dsp#4]
### Collect statistics 
* weight histograms
* over training epochs
### Explain
* why BatchNorm →  OK
* why LayerNom → NG
